message GarageMetaData {
    optional int32 doorCount = 1;
}
